deployDev.sh

```diff
@@ -0,0 +1,30 @@
+#!/bin/bash
+
+#
+# Copyright Elasticsearch B.V. and/or licensed to Elasticsearch B.V. under one
+# or more contributor license agreements. Licensed under the Elastic License;
+# you may not use this file except in compliance with the Elastic License.
+#
+
+set -e
+set +x
+
+# Expected env variables:
+# * GPROJECT - e.g. "elastic-ems"
+# * GCE_ACCOUNT - credentials for the google service account (JSON blob)
+
+if [[ -z "${GPROJECT}" ]]; then
+    echo "GPROJECT is not set, e.g. 'GPROJECT=elastic-bekitzur'"
+    exit 1
+fi
+
+export EMS_PROJECT="feature-layers"
+
+export TILE_HOST="tiles.maps.elastic.co"
+export VECTOR_HOST="storage.googleapis.com/${GPROJECT}-${EMS_PROJECT}-vector"
+export CATALOGUE_BUCKET=${GPROJECT}-${EMS_PROJECT}-catalogue
```

Comment:
i think you need a new set of buckets here - e.g. `-feature-catalogue` and `-feature-vector`

This also means you will need to create those two buckets, similar to https://github.com/elastic/infra/blob/master/terraform/providers/gcp/env/elastic-bekitzur/maps-file-service.tf#L92-L104